dotcom-rendering/src/web/browser/islands/onInteraction.ts

```diff
@@ -7,12 +7,12 @@
  */
 export const onInteraction = (
 	element: HTMLElement,
-	callback: (e: HTMLElement) => void,
+	callback: (target: Element) => void,
```

Comment:
Drive by semantic fix

Reply:
Why would anyone ever click on a non-html `Element`?

Reply:
Why indeed. But the type of `e.target` is an Element, I guess because you can set a click listener on anything if you really want to